artemis-server/src/main/java/org/apache/activemq/artemis/core/management/impl/ActiveMQServerControlImpl.java

```diff
@@ -1983,6 +1983,12 @@ public void forceFailover() throws Exception {
    }
 
    @Override
+   public void forceShutdown() throws Exception {
+      forceFailover();
+      System.exit(0);
```

Comment:
@johnament  We do that already. I guess that's what he wants as part of this feature.


I wouldn't call it forceShutdown... I would call it halt.